src/main/scala/io/smartdatalake/workflow/dataobject/TickTockHiveTableDataObject.scala

```diff
@@ -55,7 +55,7 @@ case class TickTockHiveTableDataObject(override val id: DataObjectId,
   private val connection = connectionId.map(c => getConnection[HiveTableConnection](c))
 
   // prepare final path and table
-  @transient private[workflow] lazy val hadoopPath = HdfsUtil.prefixHadoopPath(path, connection.map(_.pathPrefix))
+  @transient private[workflow] lazy val hadoopPath = HdfsUtil.prefixHadoopPath(path.getOrElse("."), connection.map(_.pathPrefix))
```

Comment:
It would be better if hadoopPath is an Option as well, because the default of "." is actionally not really what we want...

Reply:
Please see new commit: https://github.com/smart-data-lake/smart-data-lake/pull/126/commits/c2cc545d39dd48874a514ddead4d15969613a4ca
Replaced hadoopPath with a placeholder as discussed.